hopsworks-common/src/main/java/io/hops/hopsworks/common/dataset/util/DatasetHelper.java

```diff
@@ -13,7 +13,7 @@
  * You should have received a copy of the GNU Affero General Public License along with this program.
  * If not, see <https://www.gnu.org/licenses/>.
  */
-package io.hops.hopsworks.api.dataset.util;
+package io.hops.hopsworks.common.dataset.util;
```

Comment:
I needed to use it in the DatasetController which is in the common module and has no access to the api module. After talking to Ermias, decided to move the helper classes to common, so that I don't have to duplicate the helper code.